geomesa-quickstart-python/pyJavaClasses/SimpleFeatureType.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+#coding:utf-8
+"""
+SimpleFeatureType.py
+
+Description:
+   This function uses jnius to implement various features from the SimpleFeatureType(s)
+   java classes.
+
+Created by: Jordan Muss
+
+Creation Date: 4-4-2017
+Version:       1.0
+
+Dependencies: 
+         Public:     jnius
+         Private:   setupJnius
+
+Interfaces:
+        createSimpleFeatureType              Class to create a 'bare' simpleFeatureType from a list of attributes,
+                                                              and to perform appropriate tasks, e.g. set feature indices
+
+Updates:
+
+To Do:
+"""
+class simpleFeatureType():
+    def __init__(self, JNI):
+        """ Create the necessary java class interfaces: """
+        self.SimpleFeatureTypes = JNI.autoclass("org.locationtech.geomesa.utils.interop.SimpleFeatureTypes")
+
+    def createSimpleFeatureType(self, simpleFeatureTypeName,  attribute_list):
+        """ This will create a 'bare' simpleFeatureType from a list of attributes: """
```

Comment:
No idea. It is in the AccumuloQuickStart demo. In static SimpleFeatureType createSimpleFeatureType(String simpleFeatureTypeName); line 134:// create the bare simple-feature type
Maybe empty simpleFeatureType?